stock_location_last_inventory_date/__manifest__.py

```diff
@@ -3,7 +3,7 @@
 {
     "name": "Stock Location Last Inventory Date",
     "summary": "Show the last inventory date for a leaf location",
-    "version": "14.0.1.0.0",
+    "version": "14.0.1.0.1",
```

Comment:
```suggestion
    "version": "14.0.1.1.0",
```

Reply:
I would even put it as a major change
```suggestion
    "version": "14.0.2.0.0",
```

Reply:
done